Collections/Map Utilities/map.alias

```diff
@@ -178,7 +180,9 @@ if c:
   for attack in combatant.attacks:
    if attack.name == 'map':
     mapattach=combatant
-    mapinfo=attack.raw.automation[0].text
+    mapattack = mapattach.get_effect("map")
+    if attack.raw.automation:
```

Comment:
Added this so if user clears everything off the map and there is no description in the attack the map still loads